### PR TITLE
Fixes #19887 - Disable Style/ParenthesesAroundCondition cop

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ParenthesesAroundCondition - I think parentheses increase understandability.

:+1: for disabling - and thus merging.
:-1: for enabling

File: test/controllers/api/v2/fact_values_controller_test.rb [hunk 1]
```diff
@@ -11,23 +11,23 @@ def setup
     get :index
     assert_response :success
     fact_values = ActiveSupport::JSON.decode(@response.body)['results']
-    expected_hash = {@host.name=>{"kernelversion"=>"2.6.9"}}
+    expected_hash = {@host.name => {"kernelversion" => "2.6.9"}}
```
Comment (ares) [inline test/controllers/api/v2/fact_values_controller_test.rb]: are changes in this file related? (I like them :)

Comment (mmoll) [reply test/controllers/api/v2/fact_values_controller_test.rb]: @ares that's the first of the two commits.

**Review by tbrisker — APPROVED** [review]
Looks like most people prefer disabling it, and even if not, there's no consensus on enabling so I guess disable it is :)

**Pull request merged**